source/release-notes/1.4.txt

```diff
@@ -27,7 +25,57 @@ Starting in 1.4.0, you can set the workload level that
 destination clusters. For more information, see the
 :option:`--loadLevel` option.
 
+Improved Resiliency
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` 1.4.0 improves :term:`oplog` handling. The ``oplog``
+handling improvement adds resiliency to long running migrations.
```

Comment:
@davemungo We had a close-out review meeting for this project earlier today and this point came up (apologies for not getting around to replying to the now-outdated comment on this).

I think we can accentuate the problem this is meant to address and provide a high-level overview of the mechanism, since it's one of the largest pain points customers currently experience and TSEs be curious about how this works. Something like this seems reasonable to @tfogo and myself:

> Mongosync 1.4.0 adds resiliency to oplog rollover.
> 
> Previously, mongosync waited for all of the initial data to be copied before it replicated any changes to that data. For a migration with a large initial data set, this wait period made mongosync vulnerable to losing its position in the oplog, requiring the migration to be restarted from scratch.
> 
> Now, mongosync can simultaneously copy the initial data set and apply changes to that data. This means that mongosync can maintain a recent position in the oplog and that the risk of migrations having to be restarted from scratch due to large initial data sets is greatly mitigated.
> 
> See [oplog sizing](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/oplog-sizing/) for details on monitoring your migration's risk of oplog rollover.

(we're planning to update that documentation linked at the end)

Happy to go over if you think this level of detail is suitable, needs adjustments, etc.

Reply:
Updated. 

There are some concepts here that we don't really explain in much detail (eg 'initial data sync', the oplog). I added a brief intro sentence for now, do we need to expand the concept pages after the release?

I removed references to 'migration' since I think this applies to the sync use case too. Let me know if I should revert to 'migration'.

I changed the wording a little to better match our style guide. Let me know if I inadvertently changed the substance.